sim/midas/src/main/scala/midas/passes/MidasTransforms.scala

```diff
@@ -89,6 +89,7 @@ private[midas] class MidasTransforms extends Transform {
       new fame.EmitFAMEAnnotations("post-fame-transform.json"),
       new ResolveAndCheck,
       fame.MultiThreadFAME5Models,
+      fame.ImplementThreadedSyncReadMems,
```

Comment:
Could you put a comment here explaining the introduction of non-standard IR nodes and how they are handled by this pass ? Or maybe it makes sense to bundle these into a single wrapper SeqTransform?

Reply:
I changed it from a transform to just a utility, so now it's all contained within a single transform.